packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -1017,11 +1017,15 @@ class _RenderRangeSlider extends RenderBox {
   void _handleDragUpdate(DragUpdateDetails details) {
     final double dragValue = _getValueFromGlobalPosition(details.globalPosition);
 
-    // If no selection has been made yet, use the velocity of the drag to
-    // determine which thumb should be selected.
+    // If no selection has been made yet, test for thumb selection again now
+    // that the value of dx is non-zero. If this is the first selection of the
```

Comment:
"now that the value of dx is non-zero" implies that it's always non-zero; but it definitely can be zero. Are we missing an assert here, or something?

Reply:
changed to "can be"